scripts/tests/cloud-docker-credentials-integration.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -eu
+
+earthly=${earthly:=earthly}
+earthly=$(realpath "$earthly")
+echo "running tests with $earthly"
+
+# prevent the self-update of earthly from running (this ensures no bogus data is printed to stdout,
+# which would mess with the secrets data being fetched)
+date +%s > /tmp/last-earthly-prerelease-check
+
+# ensure earthly login works (and print out who gets logged in)
+test -n "$EARTHLY_TOKEN"
+"$earthly" account login
+
+# A username / password has been stored in the cloud to a docker hub user (that is not part of earthly) via:
+#   earthly secret --org earthly-technologies --project core-test-cloud-docker-credentials-test set std/registry/registry-1.docker.io/username verygoodusername
+#   earthly secret --org earthly-technologies --project core-test-cloud-docker-credentials-test set std/registry/registry-1.docker.io/password verygoodpassword # just kidding
+#
+# And our earthly-user that GHA uses has been configured to have access:
+#   earthly projects --org earthly-technologies --project core-test-cloud-docker-credentials-test members add other-service+earthly@earthly.dev read+secrets
+
+# sanity check that secrets is working
```

Comment:
https://en.wikipedia.org/wiki/Sanity_check is a valid term -- it's just a quick test to ensure things are working before proceeding (and also helps as provide a base reference if one is debugging a problem backwards). Either or seems fine to me.

Reply:
Yeah, it's just a bit archaic and ableist.  From the link you provided:
> Some software projects, such as [Android](https://en.wikipedia.org/wiki/Android_(operating_system))[[1]](https://en.wikipedia.org/wiki/Sanity_check#cite_note-1), [MediaWiki](https://en.wikipedia.org/wiki/MediaWiki)[[2]](https://en.wikipedia.org/wiki/Sanity_check#cite_note-2) and [Twitter](https://en.wikipedia.org/wiki/Twitter)[[3]](https://en.wikipedia.org/wiki/Sanity_check#cite_note-3), discourage use of the phrase sanity check in favour of just test or soundness check, as part of a wider attempt to avoid [ableist](https://en.wikipedia.org/wiki/Ableist) language and increase [inclusivity](https://en.wikipedia.org/wiki/Inclusivity).
Not a huge deal, though.

Reply:
ah interesting point! I hadn't considered that.